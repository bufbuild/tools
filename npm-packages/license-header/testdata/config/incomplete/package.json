{
  "licenseHeader": {
    "licenseType": "apache",
    "copyrightHolder": "2023"
  }
}
