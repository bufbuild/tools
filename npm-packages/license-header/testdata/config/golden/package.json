{
  "licenseHeader": {
    "licenseType": "apache",
    "yearRange": "2023",
    "copyrightHolder": "2023"
  }
}
